R/mmetrics.R

```diff
@@ -57,50 +59,48 @@ define <- function(...){
 add <- function(df, ..., metrics = ad_metrics, summarize = TRUE){
   group_vars <- rlang::enquos(...)
 
-  if(summarize){
+  if (summarize) {
     gsummarize(df, !!!group_vars, metrics = metrics)
-  } else{
+  } else {
     gmutate(df, !!!group_vars, metrics = metrics)
   }
 }
 
 #' @rdname add
 #' @export
-gsummarize <- function(df, ..., metrics){
-  group_vars <- rlang::enquos(...)
-  metrics <- mfilter(metrics, df)
-  df %>%
-    dplyr::group_by(!!!group_vars) %>%
-    dplyr::summarise(!!!metrics) %>%
-    dplyr::ungroup()
-}
+gsummarize <- function(df, ..., metrics) dplyr::summarise %>% gprocess(df, ..., metrics = metrics)
```

Comment:
We (in R) usually use `%>%` operator for `data.frame` object for easy understanding of the data flow.
That is why we hardly use `%>%` operator for function.

Could you write simply like `gprocess(dplyr::summarise, df, ..., metrics = metrics)` ?